PrebidMobile/AdUnits/Native/NativeAd.swift

```diff
@@ -296,6 +296,21 @@ import UIKit
                 Log.debug("Could not open click URL: \(clickUrl)")
             }
         }
+        if let clickTrackers = nativeAdMarkup?.link?.clicktrackers {
+            fireClickTrackers(clickTrackersUrls: clickTrackers)
+        }
+    }
+    
+    
+    private func fireClickTrackers(clickTrackersUrls: [String]) {
+        if clickTrackersUrls.count > 0 {
+            TrackerManager.shared.fireTrackerURLArray(arrayWithURLs: clickTrackersUrls) { [weak self] _ in
+                guard self != nil else {
```

Comment:
It looks like this code does not perform any meaningful job. Let's leave the completion handler empty.